### PR TITLE
Improved welcome page

### PR DESCRIPTION
**Pull request by nadonomy**
Overall improved the look, feel and content of the welcome page added in #4289. Specifically:

- Added doctype/html language declarations etc
- Added accessibility tags
- Uses a [system font stack](https://css-tricks.com/snippets/css/system-font-stack/) which includes flavours of Linux
- Added matrix logo (using in-line SVG)
- Scales on mobile etc
- Iterated on the content

Looks like:

screenshot 2018-12-13 at 13 28 04

### CODE REVIEW

File: synapse/static/index.html [hunk 1]
```diff
@@ -1,26 +1,63 @@
...
+                  </g>
+              </g>
+          </g>
+      </svg>
+    </div>
+    <h1>It works! Synapse is running</h1>
+    <p>Your Synapse server is listening on this port and is ready for messages.</p>
+    <p>To use this server you'll need <a href="https://matrix.org/docs/projects/try-matrix-now.html#clients" target="_blank">a Matrix client</a>.
+    </p>
+    <p>Welcome to the matrix universe :)</p>
```
Comment (jryans) [inline synapse/static/index.html]: Should "matrix" have a capital M here?

Comment (nadonomy) [reply synapse/static/index.html]: Good catch! Fixing.

**Comment by ara4n** [comment]
lgtm!

**Comment by nadonomy** [comment]
> (n.b. for whoever merges this that it'll need to land on the right combo of branches rather than hotfixing 0.34-release)

Aha. I just PR'd to `release-v0.34.0` as  #4289 did.

If it's easiest for me to PR again lemme know where to point to.

**Comment by hawkowl** [comment]
I'll land it into develop and then cherry-pick it into the release branch.

**Comment by richvdh** [comment]
> Aha. I just PR'd to release-v0.34.0 as #4289 did.

this was fine, for the record. I don't know why the base of the PR needed changing.

**Pull request merged**